docs/advanced-application/quiz-toolkit.md

```diff
@@ -0,0 +1,189 @@
+---
+sidebar_label: "Quiz Toolkit"
+sidebar_position: 1
+---
+# Quiz Toolkit
+
+| :trophy: This is a [premium](https://mecsimcalc.com/pricing) feature |
+| -------------------------------------------------------------------- |
+
+**MecSimCalc** now supports online quizzes.
+In this tutorial, we'll learn to build a Quiz App for educational purposes. The app displays scores 
+instantly, saves results to **Google Sheets**, and **sends email** to receivers(professors), streamlining 
+feedback and record-keeping.
+
+Seamlessly integrates with **[Google Sheets](google-sheet)** for straightforward and organized data compilation.
+![img alt](/docs/advanced-application/quiz-toolkit/sheet_res.png)
+
+
+Utilizes **[Gmail](gmail)** for direct communication and data sharing, ensuring immediate feedback and results dissemination.
+![img alt](/docs/advanced-application/quiz-toolkit/gmail_res.png)
+
+:::info
+You will need to provide additional parameters for these two features. **Don't worry, we will teach you how to obtain them later!**
+:::
+## Steps 1: Create your quiz App
+For example, you can get student's `name` and `id` from the input, and calculate `grade` through code. If you want this information to be exported to your own Google Sheet, you can do it like this:
+```python
+"""
+Assume that inputs['name'] = "Joe", inputs['id'] = "186234", inputs['grade'] = 85
+"""
+import mecsimcalc as msc
+
+def main(inputs):
+    grade = ... #your logic here
+    
+    #values you want to collect
+    values = [
+    [ inputs['name'], inputs['id'], grade ],
+    ]    
+
+    # For Google Sheet:
+    service_account_info = {
+        # We will teach you how to get this later, don't worry
+    }
+    spreadsheet_id = 'your_spreadsheet_id_here'
+    msc.send_gmail(sender_email, receiver_email, subject, app_password, values)
```

Comment:
The send_gmail & append_to_google_sheet are swapped with each other in the code (incorrect places)